pkg/skeletor/mixin.go

```diff
@@ -5,15 +5,20 @@ import (
 	"get.porter.sh/porter/pkg/context"
 )
 
+const defaultClientVersion string = "v0.0.0"
+
 type Mixin struct {
 	*context.Context
+	ClientVersion string
 	//add whatever other context/state is needed here
 }
 
 // New azure mixin client, initialized with useful defaults.
 func New() (*Mixin, error) {
 	return &Mixin{
-		Context: context.New(),
+		Context:       context.New(),
+		ClientVersion: defaultClientVersion,
+		//add whatever is needed here
```

Comment:
nit: I wonder if the similar note in the `Mixin` struct declaration is enough?  Perhaps we don't need the same note here as well?